Src/Web/Web.Nuget/Package.nuspec

```diff
@@ -19,7 +19,7 @@
       <group targetFramework="net45">
         <dependency id="Microsoft.ApplicationInsights" version="[$coresdkversion$]" />
         <dependency id="Microsoft.ApplicationInsights.WindowsServer" version="$version$" />
-        <dependency id="Microsoft.AspNet.TelemetryCorrelation" version="1.0.1" />
+        <dependency id="Microsoft.AspNet.TelemetryCorrelation" version="1.0.2" />
```

Comment:
nuspec and packages config should be aligned on the required version of this package